### PR TITLE
Sunset Nearby 

### PR DESCRIPTION
**Pull request by sharvaniharan**
Removed Fragment
Removed option in NavTab
Removed analytics for Nearby
Removed tests
Removed xml resources

**phab**: https://phabricator.wikimedia.org/T228661

### CODE REVIEW

**Comment by dbrant** [comment]
@sharvaniharan This should now be mergeable. Can we bring it up to date?

**Comment by sharvaniharan** [comment]
Updated @dbrant .

File: app/src/main/java/org/wikipedia/analytics/SessionFunnel.java [hunk 1]
```diff
@@ -104,7 +104,6 @@ private void logSessionData() {
                 "fromExternal", sessionData.getPagesFromExternal(),
                 "fromHistory", sessionData.getPagesFromHistory(),
                 "fromReadingList", sessionData.getPagesFromReadingList(),
-                "fromNearby", sessionData.getPagesFromNearby(),
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/analytics/SessionFunnel.java]: Unfortunately, `fromNearby` is a required field in the eventlogging schema. Let's leave this field here, but just give it a value of `0`.

Comment (sharvaniharan) [reply app/src/main/java/org/wikipedia/analytics/SessionFunnel.java]: Wow! awesome catch! - Done @dbrant

File: app/src/main/java/org/wikipedia/navtab/NavTab.java [hunk 2]
```diff
@@ -34,11 +33,6 @@
         @NonNull @Override public Fragment newInstance() {
             return SuggestedEditsTasksFragment.Companion.newInstance();
         }
-    },
-    NEARBY(R.string.nav_item_nearby, R.drawable.ic_explore_black_24dp) {
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/navtab/NavTab.java]: Should we remove `ic_explore_black_24dp`?

**Pull request merged**